### ISSUE TITLE
Allow events to span across days while under eventConstraint

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#2677]
Originally reported on Google Code with ID 2412
```
It would be nice to allow events to span days even when there is a eventConstraint defined.

Currently, you can't span an event across more than one day when the eventConstraint
is defined.

It's great for eventConstraint to disallow creating, editing, dropping or moving events
outside of the defined eventContraint parameters, it helps prevent scheduling errors
by staff members.  However, events should be allowed to span days within the eventConstraint
as long as the event start and end times are less than or equal to the start and end
parameters of the eventConstraint option.

A company with 9 to 5 business hours 7 days a week still needs to be able to schedule
events that span more than one day.

thanks,



```

Reported by `nelsonxm56` on 2015-01-11 23:53:32

Imported with **1** stars.

**Comment by Rastalamm** [comment]
So does this mean a new event that is constrained by a background event can't be a multiday event? Is that why I can't set the 11:30pm time slot?

**Comment by arshaw** [comment]
@Rastalamm you'd have to post a [JSBin](http://fullcalendar.io/support/) for me to fully understand what you are doing, but this ticket is about making it so that all-day events are not constrained by eventContraint if eventContraint happens to be timed (I believe)

**Comment by Rastalamm** [comment]
@arshaw I will get something up within a few days but pretty much I have a background event from 12 pm until 12am on Wednesday. I am using event constraint on that event and when I try to create a new event (on top of the constrained one) I can't create anything in the last 30 min time slot (11:30pm-12:00am). I thought it had something to do with this issue as the new event is technically "crossing into" the next day.

**Comment by Rastalamm** [comment]
@arshaw - I ended up figuring it out. 

I am splitting events (from 1 to 2) when they span multi days when a user crosses timezones. Those events are all constraint events and are being rendered in the background. 

When I was splitting them, I was setting the end time to moments endOf('day') method. This was causing the end time to be "23:59". The new events (30 min slots) were not fully constrained, thus not allowing me to create them. I ended up forcing the end to be "24:00" if they matched the "23:59" time format.
